src/app/components/utility/modal/modal.component.ts

```diff
@@ -14,125 +13,102 @@ import { AuthService } from '../../../services/auth.service';
   styleUrls: ['./modal.component.scss']
 })
 export class ModalComponent implements OnInit {
-
   /**
    * Input parameters object
    */
   @Input() params: any;
-
```

Comment:
Please revert this and similar changes.

Reply:
but sir there are deleted lines you said me to revert